src/documentation/nl_syntus_fares_read.md

```diff
@@ -0,0 +1,59 @@
+
+### Ticket
+NTM Property | Source frame | Source element | Notes/Mapping rule
+--- | --- | --- | ---
+id | *FareFrame* | *DistanceMatrixElement{id}* | 
+input_data_format | | | Fixed value `nl_syntus_fares`.
+start_date | *ResourceFrame* | *versions/Version/StartDate* | 
+end_date | *ResourceFrame* | *versions/Version/EndDate* | 
+currency_type | *FareFrame* | *FrameDefaults/DefaultCurrency* | 
+price | | | See the mapping rule below.
+
+**Computing the ticket price**
+
+The ticket price is calculated by adding the boarding fee to the price specified for the origin-destination pair:
+- the boarding fee is equal to the value of *FareFrame/EntranceRateWrtCurrency*.
+- the OD price is calculated by multiplying the value of *DistanceMatrixElementPrice/Amount* by the value of *DistanceMatrixElementPrice/Units*.
+
+If *FareFrame/RoundingWrtCurrencyRule* is specified, a rounding rule for the specified `currency_type` is applied to the computed ticket price. For example, if the value is set to `0.01` for the currency `EUR`, then the ticket price is rounded to the nearest euro cent.
+
+If the computed ticket price exceeds the value of *FareFrame/CappingWrtCurrencyRule*, then the latter is taken into account.
+
+### OD Rules
+NTM property | Source frame | Source element | Notes/Mapping rule
+--- | --- | --- | ---
+id | *FareFrame* | *DistanceMatrixElement{id}* | The id is prefixed with `OD:`.
+ticket_id | | | Id of the `Ticket` to which this `OD Rule` is applied.
+origin_stoparea_id | *FareFrame* | *DistanceMatrixElement/StartStopPointRef{ref}* | See the mapping rule below.
+dest_stoparea_id | *FareFrame* | *DistanceMatrixElement/EndStopPointRef{ref}* | See the mapping rule below.
+
+**Setting the origin_stoparea_id and the dest_stoparea_id of an OD Rule**
+
+Finding the right stops in the NTFS to which a fare is applied is not straightforward. The stops in the *FareFrame* point to the *ScheduledStopPoint*s in the *ServiceFrame*. The *ScheduledStopPoint*s are composed of *PointProjection*s that are referenced in the NTFS.
+
+The *origin_stoparea_id* should have an associated stop_point with a complementary code of type `gtfs_stop_code` that matches the value of *ProjectedPointRef{ref}* (without the network prefixe, if any) of the *ScheduledStopPoint* in the *ServiceFrame* whose *id* is referenced by *StartStopPointRef{ref}* in the *DistanceMatrixElement*.
+
+The *dest_stoparea_id* should have an associated stop_point with a complementary code of type `gtfs_stop_code` that matches the value of *ProjectedPointRef{ref}* (without the network prefixe, if any) of the *ScheduledStopPoint* in the *ServiceFrame* whose *id* is referenced by *EndStopPointRef{ref}* in the *DistanceMatrixElement*.
+
+If no matching is found for the origin or the destination stop, then the stop is ignored and no rule is created.
```

Comment:
In addition to the rule, the ticket corresponding to this specific OD should not be created in this case neither.